src/core/api/public_api.ts

```diff
@@ -1865,6 +1870,15 @@ class Player extends EventEmitter<IPublicAPIEvent> {
     this._priv_bufferOptions.wantedBufferAhead.setValue(sizeInSeconds);
   }
 
+  /**
+   * Set the max buffer size the buffer should take in memory
+   * The player . will stop downloading chunks when this size is reached.
+   * @param {Number} sizeInBits
+   */
+  setmaxVideoBufferSize(sizeInBits : number) : void {
```

Comment:
/!\ we follow camelCase syntax (as usual in JS/TS land) so it should be `setMax` not `setmax`